### PR TITLE
Unreal spatial audio doc

### PR DESCRIPTION
**Pull request by hferrone**

### CODE REVIEW

File: mixed-reality-docs/unreal-spatial-sound.md [hunk 1]
```diff
@@ -54,15 +54,36 @@ Spatial audio is disabled by default on desktop versions of Windows. You can ena
 >This setting is only required if you plan to test your project in the Unreal editor.
 
 ## Creating Attenuation objects
-After you've installed and configured the necessary plugins you can create a new Sound Attenuation asset by:
-*  Right-clicking on the **Content Browser** and selecting **Create Advanced Asset -> Sounds -> Sound Attenuation**:
+After you've installed and configured the necessary plugins:
+* Place an **Ambient Sound actor** in the scene from the **Place Actors** window.
+
+    // TODO: What visual qualities? How is this connected to spatial audio?
+    * Once the Ambient Sound is created, the actor can be attached to a parent, which has visual qualities.
```
Comment (NoelCross) [inline mixed-reality-docs/unreal-spatial-sound.md:61]: Visual qualities is probably not the right term.  What I'm trying to express is that an Ambient Sound actor doesn't have any visuals by default.  You'll just hear a sound from the place in the scene and you can't see anything.  By making an Ambient Sound a child of a visual element, then when the visual element moves in the scene, the sound will also move in the scene too.

File: mixed-reality-docs/unreal-spatial-sound.md [hunk 2]
```diff
@@ -54,15 +54,36 @@ Spatial audio is disabled by default on desktop versions of Windows. You can ena
...
 
 ![Creating sound attenuation asset](images/unreal-spatial-audio-img-06.png)
 
-Your new sound asset will be created with a **Details** and all the spatial sound settings you could ask for:
+* Then switch the **Spatialization Method** to **Binaural**.
 
 ![New sound attenuation asset](images/unreal-spatial-audio-img-03.png)
 
+Putting it all together:
+* Add the sound attenuation asset and the attenuation settings to the **Ambient Sound actor** in yoru scene:
```
Comment (NoelCross) [inline mixed-reality-docs/unreal-spatial-sound.md:75]: yoru - typo

File: mixed-reality-docs/unreal-spatial-sound.md [hunk 3]
```diff
@@ -54,15 +54,36 @@ Spatial audio is disabled by default on desktop versions of Windows. You can ena
...
+
+// TODO: Add image for this step
+
+// TODO: Clarify the instruction below
+
+The Ambient Sound actor has properties for setting a Sound Attenuation setting as well as the sound asset to use for the Ambient Sound actor.
+
+// TODO: How do you do this? What is the hardware offload support? More details...
```
Comment (NoelCross) [inline mixed-reality-docs/unreal-spatial-sound.md:83]: There's a property on the Ambient Sound actor for specifying the attenuation settings object.

image

File: mixed-reality-docs/unreal-spatial-sound.md (hunk 1 above)
Comment (hferrone) [reply mixed-reality-docs/unreal-spatial-sound.md:61]: Perfect.

File: mixed-reality-docs/unreal-spatial-sound.md (hunk 2 above)
Comment (hferrone) [reply mixed-reality-docs/unreal-spatial-sound.md:75]: Fixed.

File: mixed-reality-docs/unreal-spatial-sound.md (hunk 3 above)
Comment (hferrone) [reply mixed-reality-docs/unreal-spatial-sound.md:83]: I don't think we can use this image with the red line. If you post the original screenshot, I can add a callout box to match our formatting.

Comment (NoelCross) [reply mixed-reality-docs/unreal-spatial-sound.md]: image

Here's a cleaned up version of the image.

**Comment by hferrone** [comment]
@NoelCross If you could take one last look then approve the changes we can get this published.

File: mixed-reality-docs/unreal-spatial-sound.md (hunk 3 above)
Comment (hferrone) [reply mixed-reality-docs/unreal-spatial-sound.md]: Perfect.

File: mixed-reality-docs/unreal-spatial-sound.md [hunk 4]
```diff
@@ -54,15 +54,31 @@ Spatial audio is disabled by default on desktop versions of Windows. You can ena
 >This setting is only required if you plan to test your project in the Unreal editor.
 
 ## Creating Attenuation objects
-After you've installed and configured the necessary plugins you can create a new Sound Attenuation asset by:
-*  Right-clicking on the **Content Browser** and selecting **Create Advanced Asset -> Sounds -> Sound Attenuation**:
+After you've installed and configured the necessary plugins:
+* Search for an **Ambient Sound** actor in the **Place Actors** window and hit Enter to place it in the scene.
```
Comment (NoelCross) [inline mixed-reality-docs/unreal-spatial-sound.md:58]: Enter won't do anything from the Place Actors window.  You need to drag the Ambient Sound actor from the Place Actors window into the scene window in order place the sound object in the scene.

File: mixed-reality-docs/unreal-spatial-sound.md [hunk 5]
```diff
@@ -54,15 +54,31 @@ Spatial audio is disabled by default on desktop versions of Windows. You can ena
...
+* Make the **Ambient Sound** actor a child of a visual element in your scene. 
+    * An Ambient Sound actor doesn't have any visual representation by default, so you'll only hear a sound from its position in the scene. Attaching it to a visual element let's you see and move the actor like any other asset.
+
+To create a sound attenuation asset:
+*  Right-click on the **Content Browser** and selecting **Create Advanced Asset -> Sounds -> Sound Attenuation**:
 
 ![Creating sound attenuation asset](images/unreal-spatial-audio-img-06.png)
 
-Your new sound asset will be created with a **Details** and all the spatial sound settings you could ask for:
+* Then switch the **Spatialization Method** to **Binaural**.
```
Comment (NoelCross) [inline mixed-reality-docs/unreal-spatial-sound.md:70]: You set the spatialization mode by "editing" the SoundAttenuation asset.   To edit, right click on the SoundAttenuation asset in the Content Browser window and select the Edit... option to bring up the properties window.

File: mixed-reality-docs/unreal-spatial-sound.md [hunk 6]
```diff
@@ -54,15 +54,31 @@ Spatial audio is disabled by default on desktop versions of Windows. You can ena
...
 
 ![New sound attenuation asset](images/unreal-spatial-audio-img-03.png)
 
+Putting it all together:
+* Select the **Ambient Sound** actor and scroll down to the **Attenuation** section in the **Details** panel. 
+    * Set the **Attenuation Settings** property to the **Sound Attenuation** asset you created.
+
+![New sound attenuation asset](images/unreal-spatial-audio-img-08.png)
+
+Once all of this is configured the ambient sound can be spatialized using the dedicated hardware offload support on HoloLens 2.
```
Comment (NoelCross) [inline mixed-reality-docs/unreal-spatial-sound.md:80]: The next step is to set the SoundAsset you want to attach to the Ambient Sound actor.  Do this by updating the properties of the Ambient Sound actor to specify the SoundAsset file to use.

image

Note: this SoundAsset file needs to be mono in order to spatialize via the Microsoft Spatial Sound plug-in.  The properties of the sound file can be observed by hovering over the asset in the Content Browser window.

image

**Comment by hferrone** [comment]
@NoelCross Give this a look and if you approve of the changes we can push it live.

File: mixed-reality-docs/unreal-spatial-audio.md [hunk 7]
```diff
@@ -55,36 +55,41 @@ Spatial audio is disabled by default on desktop versions of Windows. You can ena
...
 
-![New sound attenuation asset](images/unreal-spatial-audio-img-08.png)
+![Set attenuation setting](images/unreal-spatial-audio-img-08.png)
+
+6. Set the **Sound Asset** you want to attach to the Ambient Sound actor by updating the **Sound** property of the Ambient Sound actor to specify the SoundAsset file to use.
+
+![Set sound asset](images/unreal-spatial-audio-img-09.png)
+
+> [!NOTE] 
+> The SoundAsset file needs to be mono to be spatializes with the Microsoft Spatial Sound plug-in. You can find the sound file properties by hovering over the asset in the Content Browser window as shown in the screenshot below.
```
Comment (NoelCross) [inline mixed-reality-docs/unreal-spatial-audio.md:84]: "spatializes" should be replaced with "spatialized"

Comment (hferrone) [reply mixed-reality-docs/unreal-spatial-audio.md]: Good catch.

**Comment by hferrone** [comment]
@NoelCross This good to go?

**Pull request merged**